src/app/api/payment/route.ts

```diff
@@ -84,3 +88,31 @@ export async function POST(request: Request) {
         return new Response(null, { status: 500 });
     }
 }
+
```

Comment:
Can there be a comment on what this function does? (i.e. updating the expiry date if the admin has authorised it)

It doesn't need to be long, one line is fine